ecs-init/Godeps/Godeps.json

```diff
@@ -15,8 +126,78 @@
 			"Rev": "c9ad0ce23f68428421adfc6ced9e6123f54788a5"
 		},
 		{
+			"ImportPath": "github.com/fsouza/go-dockerclient/external/github.com/Sirupsen/logrus",
```

Comment:
This was the same go-dockerclient version. There is a known bug with Godep v79 that vendors nested dependencies that have already been vendored. (https://github.com/tools/godep/issues/498)

Reverting to Godep v29 (latest version used to ecs-init) fixes this. Update pending.

Additionally, the latest versions of go-dockerclient do not support go 1.5. I would like to get both of these dependencies up to date and migrate to the vendor/ folder in a future change, but it is out of scope for this PR.